src/components/profile/CreateAccountForm.tsx

```diff
@@ -31,15 +30,26 @@ const CreateAccountForm = () => {
 
   const handleCreateAccountButtonClick = () => {
     if (isValid()) {
-      createAccountWithEmailandPassword(
-        username,
-        email,
-        password,
-        confirmPassword
-      )
+      createUserWithEmailAndPassword(email, password)
     }
   }
-
+  if (error) {
```

Comment:
where are these `error`, `loading`, and `user` variables coming from? I don't see a `useAuthState` import. Also because this is a child component of `SignInTabs` which does include `useAuthState` none of these variables should be needed here.